backend/liveness/liveness.go

```diff
@@ -245,15 +246,23 @@ func (t *SwitchSet) monitor(ctx context.Context) {
 		}
 	}()
 	go func() {
+		var wc clientv3.WatchChan
+		ctx := clientv3.WithRequireLeader(ctx)
+		limiter := rate.NewLimiter(rate.Every(time.Second), 1)
+		_ = limiter.Wait(ctx)
+	OUTER:
+		wc = t.client.Watch(ctx, t.prefix, clientv3.WithPrefix(), clientv3.WithPrevKV())
```

Comment:
Stupid question: do we need to move the rate limiter `Wait` method within the `OUTER` label? Maybe I'm misreading it because what I understand is that it would only be rate limiting once the goroutine starts.

Reply:
Ah good question. The intent of the limiter is that no more than one `Watch` request will be made per second. That is, if the system tries to create a `Watch` over and over again, but fails consistently, it will not do so faster than once per second.